### PR TITLE
Add more fields to accounts in graphql

### PR DESCRIPTION
**Pull request by deepthiskumar**
Added permissions, verification key, sequence state

Checklist:

- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #0000

### CODE REVIEW

File: src/lib/mina_graphql/mina_graphql.ml [hunk 1]
```diff
@@ -1213,11 +1291,36 @@ module Types = struct
...
+             ; field "verificationKey" ~typ:string
+                 ~doc:
+                   "Hash of the verification key associated with this account"
+                 ~args:Arg.[]
+                 ~resolve:(fun _ { account; _ } ->
+                   Option.value_map account.Account.Poly.snapp ~default:None
+                     ~f:(fun snapp_account ->
+                       Option.map snapp_account.verification_key ~f:(fun vk ->
+                           With_hash.hash vk
+                           |> Pickles.Backend.Tick.Field.to_string)))
```
Comment (mrmr1993) [inline src/lib/mina_graphql/mina_graphql.ml:1313]: Shouldn't this use the encoding for verification keys, to allow for matching the input against a potential snapp?

Comment (deepthiskumar) [reply src/lib/mina_graphql/mina_graphql.ml:1313]: This is just the hash of the verification key which is good enough to check if vk was updated correctly. Do we want to show verification key as well?

**Review by psteckler — APPROVED** [review]

File: src/lib/mina_graphql/mina_graphql.ml (hunk 1 above)
Comment (mrmr1993) [reply src/lib/mina_graphql/mina_graphql.ml:1313]: It seems odd that you set it with a transaction that contains the actual vk but then only get to confirm via the hash. We should probably have both.

Comment (deepthiskumar) [reply src/lib/mina_graphql/mina_graphql.ml:1313]: The transaction contains both the key and its hash https://github.com/MinaProtocol/mina/blob/c7c4c379e98c6078cd6122e1671099c66a966616/src/lib/mina_graphql/mina_graphql.ml#L2213

**Review by MartinMinkov — APPROVED** [review]

File: src/lib/mina_graphql/mina_graphql.ml (hunk 1 above)
Comment (mrmr1993) [reply src/lib/mina_graphql/mina_graphql.ml:1313]: Sure, but if you don't think to query it, and later want to check that the verification key was updated correctly, you have no way to get that hash.

Comment (deepthiskumar) [reply src/lib/mina_graphql/mina_graphql.ml:1313]: Fair enough. I'll add vk as well

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**